### PR TITLE
feat[WIP]: Login and signup screens

### PR DESCRIPTION
**Pull request by saadahmadhere**
This PR consists of adding login and signup screens.

### CODE REVIEW

**Review by omjadhav85 — APPROVED** [review]
UI looks good and clean. Good to merge!

File: src/screens/Login/Login.jsx [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+            Password
+          </label>
+          <input
+            type="password"
+            id="password"
+            className="input block"
+            placeholder="Your password here..."
+            required
+          />
+          <a className="btn_icon btn_show__password">
```
Comment (Tanay-J) [inline src/screens/Login/Login.jsx]: No need of anchor tag here

File: src/screens/Signup/signup.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+.signUp input {
+  margin-bottom: var(--size-4);
+}
+
+.signUp .btn {
+  align-self: center;
+  width: 100%;
+}
+
+.signUp a {
```
Comment (Tanay-J) [inline src/screens/Signup/signup.css]: I guess this is an unused class, you may remove it

**Review by Tanay-J — APPROVED** [review]
Code looks neat. I like how you have used semantic html tags at many places.
Made a couple of suggestions, you may have a look.
Approving the changes.

File: src/screens/Login/Login.jsx (hunk 1 above)
Comment (saadahmadhere) [reply src/screens/Login/Login.jsx]: Thanks for pointing that out, fixed in https://github.com/saadahmadhere/Wisdom-library/pull/4/commits/023da4d2dc191103e163c209ab1b3ee261d9317c

File: src/screens/Signup/signup.css (hunk 2 above)
Comment (saadahmadhere) [reply src/screens/Signup/signup.css]: fixed in https://github.com/saadahmadhere/Wisdom-library/pull/4/commits/59c52d8f239e025032b57404d0246a43d0ad4e20

**Pull request merged**